doc/getting-started/getting-started.rst

```diff
@@ -608,6 +608,30 @@ Install ACRN
    The target system will reboot into the ACRN hypervisor and
    start the Ubuntu Service VM.
 
+#. Confirm that you see the GRUB menu with the "Ubuntu-ACRN Board Inspector" entry. Select
+   it and proceed to booting ACRN. (It may be auto-selected, in which case it
+   will boot with this option automatically in 5 seconds.)
+
+   Note: Maybe there are several same items display, you can check them via pressng `e` to
```

Comment:
These multiple GRUB entries is a rather embarrassing developer experience problem including the documentation "workaround" to check each one (using the 'e'dit option).  Can there be an engineering solution to remove the extra entries? Do we know why there are duplicates for both the hypervisor and board inspector?  Why is there entries for the board inspector that weren't there for 3.1?  @NanlinXie @junjiemao1

Reply:
Have added kernel version to GRUB menuentry. please refer to #8357 and #8358 for code change.